block_producer/Cargo.toml

```diff
@@ -24,7 +24,8 @@ helper_functions = { workspace = true }
 hex-literal = { workspace = true, optional = true }
 itertools = { workspace = true }
 keymanager = { workspace = true }
-log = { workspace = true }
+tracing = { workspace = true }
+logging ={ workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order